### DISCUSSION TITLE
Platform Extension for GNSS Meteorology

### DISCUSSION THREAD

**Discussion by akvas** [groops-devs/groops#262] [Ideas]
Hi all,

I want to start a discussion about introducing a new PlatformEquipment type to the Platform file format. 

In GNSS meteorology it is generally the case to have sensors (pressure, temperature, relative humidity) co-located with the GNSS station to accurately measure the local meteorological conditions. As is the case with receivers and antennas, these sensors may be replaced over time. 

In order to properly track and document these changes, it would be useful to store these sensor changes with other platform (station) metadata. This can currently be done with the PlatformEquipment "other", however, this makes it difficult to use it in programs because it cannot be distinguished via type. 

If you agree that this might be a useful feature, I am happy to create a PR.

Cheers,
Andreas

**Comment by Waley04** [comment]
@akvas, I agree that introducing a new PlatformEquipment type for sensors would be a valuable addition. It would enhance the clarity and usability of the platform metadata.

Regards,

Waley.

**Comment by Hanbing59** [comment]
Hi Andres,

I am wondering if you have any updates on this idea.
